### PR TITLE
Annual pricing toggle

### PR DESCRIPTION
**Pull request by kilajyot**

### CODE REVIEW

**Comment by thtmnisamnstr** [comment]
Preview URL: https://6442be82b4004e22a30a12cc--ephemeral-narwhal-3ac3b2.netlify.app

File: website/assets/img/why-earthly.webp
Comment (vladaionescu) [inline website/assets/img/why-earthly.webp:1]: This image looks blurry, and also it doesn't match the instructions in the sketch. Perhaps we should use the old image for now, until we fix this.

File: website/pricing.md [hunk 1]
```diff
@@ -14,6 +14,15 @@ pageStyle: bg-background-pricing
...
 </div>
 
+<div class="flex items-center mb-4">
+  <label class="toggle-switch">
+    <input id="pricing-toggle-switch" type="checkbox" checked>
+    <span class="slider"></span>
+  </label>
+
+  <span class="ml-2">Annual Pricing</span>
+</div>
```
Comment (vladaionescu) [inline website/pricing.md:24]: Perhaps a question for @mirho - should this be centered and larger?

**Review by vladaionescu — COMMENTED** [review]
Thanks for sending this over! I left a couple of comments.

**Comment by thtmnisamnstr** [comment]
@vladaionescu @mirho The changes here are for the items in [this spreadsheet](https://docs.google.com/spreadsheets/d/1Jc8hHQs51MIT89kPRFm0QVKDbo43JW2ZFNDap7X2gaA/edit?usp=sharing) where the PR Link column has a value of #434.

File: website/pricing.md (hunk 1 above)
Comment (mirho) [reply website/pricing.md:24]: @vladaionescu this does not look right. We did not hand over a design for this element.

Let me make a mockup of what it should look like, and I will add it as a comment.

**Comment by mirho** [comment]
@kilajyot @thtmnisamnstr @vladaionescu  Here is the design mockup for the pricing toggle. 

image

**Comment by thtmnisamnstr** [comment]
@mirho That looks great.

**Comment by thtmnisamnstr** [comment]
New Preview URL: https://64480887040a7000b5d12b5f--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by mirho** [comment]
I reviewed the pricing page, and there are multiple deviations from the pricing page design. See the screenshot of the design and implementation side-by-side. 

1) There is a big gap between the pricing title and the "Note:..." that should not be there.
2) The "Note: A Free plan..." text is not the right color
3) The toggle buttons are smaller than in the design both vertically, and horizontally.
4) Annual should be the default state

Please make sure to follow the design closely when implementing these in html/css. Thanks.

image

**Comment by vladaionescu** [comment]
I spoke with @mirho and we think that it's best to first merge this PR to get things live right now, and do a follow-up PR to make the implementation consistent with the design based on his comments.

**Review by vladaionescu — APPROVED** [review]

**Comment by avinder-red-crackle** [comment]
@mirho Can you provide us with a mockup in figma for the pricing page? That will help us in implementing the accurate design.

**Pull request merged**